src/test/scala/me/rexim/github/EtagPollingSpec.scala

```diff
@@ -0,0 +1,24 @@
+package me.rexim.issuestant.github
+
+import org.scalatest._
+import me.rexim.issuestant.mock._
+
+import org.http4s._
+import org.http4s.client._
+
+class EtagPollingSpec extends FlatSpec with Matchers {
+  behavior of "EtagPolling"
+
+  it should "reuse etag header for the service response" in {
+    val etagLogging = new EtagLogging
+    val etagPolling = new EtagPolling(
+      etagLogging.client,
+      Uri(path = "/rexim")
+    )
+
+    val responses = etagPolling.responses.take(5).runLog.run.toList
```

Comment:
There is no need to Responses are generated by EtagLogging. Testing EtagLogging is a different story.